truetype/truetype.go

```diff
@@ -306,6 +314,141 @@ func (f *Font) parseMaxp() error {
 	return nil
 }
 
+func (f *Font) parseName() error {
+	if len(f.name) < 18 { // The name table is three uint16s + at least one Name Record.  Name Records are six uint16s.
+		return FormatError("name table too short")
+	}
+	if u16(f.name, 0) != 0 { // The first uint16 in the name table should be set to 0, per the spec.
+		return FormatError(fmt.Sprintf("invalid format specifier: %d", u16(f.name, 0)))
+	}
+
+	numNames := int(u16(f.name, 2)) // The number of Name Records is the second uint16 in the name table.
+	stringOffset := u16(f.name, 4)  // And the offset marking the beginning of the character data is the third int16.
+
+	if len(f.name) < int(stringOffset) {
+		return FormatError("string offset is larger than the name table")
+	}
+
+	const ( // The platform IDs in the name table.
+		PLATFORM_UNICODE   = 0
```

Comment:
Go names lookLikeThis, NOT_LIKE_THAT, even for constant names.

But it seems like you should be able to re-factor the constants like microsoftSymbolEncoding that are defined in parseCmap.

BTW it looks weird how you say that MICROSOFT_SYMBOL_CS has the value 1, but in parseCmap it says 0.